backend/buildeR.Kafka/KafkaConsumer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Confluent.Kafka;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace buildeR.SignalR.Services
+{
+    public class KafkaConsumer
+    {
+        private ConsumerConfig _consumerConfig;
+        private string _topic;
+        public IConsumer<Null, string> consumer;
+        public KafkaConsumer(string topic)
+        {
+            _consumerConfig = new ConsumerConfig
+            {
+                GroupId = "logs-consumers-group",
+                BootstrapServers = "localhost:9092",
```

Comment:
Could you move this to appsettings

Reply:
Should I create appsettings for Kafka class library?